packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -506,6 +509,12 @@ export class AzureCommunicationCallAdapter<AgentType extends CallAgent | BetaTea
     });
   }
 
+  /* @conditional-compile-remove(unsupported-browser) */
+  public allowUnsupportedBrowserVersion(): void {
+    this.context.setState({ ...this.context.getState(), unsupportedBrowserVersionsAllowed: true });
+    return this.context.updateClientState(this.callClient.getState());
```

Comment:
is `updateClientState` needed? `setState` should be set up to ensure this is called

Reply:
A little roundabout but looks like, setState, will emit onStateChange, that will then have the call adapter trigger updateClientState:
![image](https://user-images.githubusercontent.com/2684369/206801894-4bbafd52-4221-4b19-8412-a9930a877953.png)